subgraphs/kwenta/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "license": "MIT",
+  "scripts": {
+    "format": "npx prettier --write .",
+    "prepare:constants": "mustache protocols/${npm_config_protocol}/config/deployments/${npm_config_id}/configurations.json configurations/configure.mustache > configurations/configure.ts"
```

Comment:
don't need this since you've turned `"prepare:constants": true` in deployments.json